Tasks/JenkinsQueueJob/jenkinsqueuejobtask.ts

```diff
@@ -111,6 +111,9 @@ async function doWork() {
         var rootJob = await util.pollCreateRootJob(queueUri, jobQueue, taskOptions);
         //start the job queue
         jobQueue.start();
+        //store the job name in the output variable
+        tl.debug('JENKINS_JOB_ID=' + rootJob.executableNumber);
+        tl.setVariable("JENKINS_JOB_ID", rootJob.executableNumber.toString());
```

Comment:
Yes, see how the task lib already outputs debug info [here](https://github.com/Microsoft/vsts-task-lib/blob/212473e933d4d56416d1d1c91280d88c0c75c573/node/task.ts#L120).